notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb

```diff
@@ -0,0 +1,827 @@
+      "text/plain": [
+       "<Figure size 864x144 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "az.to_netcdf(data_bad_init, os.path.join(NETCDF_DIR, \"data_bad_init.nc\"))\n",
+    "az.plot_trace(data_bad_init)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this traceplot the chains have not converged to a single value. Both seem to be trending towards a value, but that value hasn't been reached. Additionally the posterior does not seem to follow any known distribution. This is concerning as we were expecting our model to exhibit a Gaussian behavior but it's hard to see one\n",
+    "In practice picking initialization points a challenge and a [number of strategies exist](https://docs.pymc.io/api/inference.html)\n",
+    "\n",
+    "#### Suggested action\n",
+    "* Try different initialization strategies\n",
+    "* Run inference across more chains in different points"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 2: No Warmup Period\n",
```

Comment:
titled "Burn-in for MCMC, why we prefer the term warm-up": 
https://statmodeling.stat.columbia.edu/2017/12/15/burn-vs-warm-iterative-simulation-algorithms/

Reply:
That is super weird to me, introducing a new term just to replace an established term just because it may lead to a "misleading analogy". For me this generally only adds confusion. I feel like this Nikolai Vetr guy  "I’ve thought burn-ins were more you waiting for the chain to reach stationary since they get initialized somewhere decidedly not in the joint posterior, where warm-ups did other stuff" and then he add "...> Rethinking: Warmup is not burn-in. " haha.

Reply:
Should I swap the words "warm up" with the words "burn in"? I'm unopinionated and happy to do whatever we think is best

Reply:
Not sure anymore, haha

Reply:
Added a note that both terms exist and linked to gelmans article